pkg/cnbutils/copy_project.go

```diff
@@ -11,23 +12,94 @@ import (
 	ignore "github.com/sabhiram/go-gitignore"
 )
 
-func CopyProject(source, target string, include, exclude *ignore.GitIgnore, utils BuildUtils) error {
-	sourceFiles, _ := utils.Glob(path.Join(source, "**"))
+func isFiltered(path string, knownSymlinks []string) bool {
```

Comment:
naming: How about `isSymlinked` or `shouldBeFiltered`?

It is not filtered yet, right?